test_settings.py

```diff
@@ -30,7 +30,28 @@
 EDX_DRF_EXTENSIONS = {}
 
 JWT_AUTH = {
+
+    'JWT_ALGORITHM': 'HS256',
+
     'JWT_AUDIENCE': 'test-aud',
-    'JWT_ISSUER': 'test-iss',
-    'JWT_SECRET_KEY': 'tell-no-one',
+
+    'JWT_DECODE_HANDLER': 'edx_rest_framework_extensions.utils.jwt_decode_handler',
+
+    'JWT_ISSUERS': [
+        {
+            'ISSUER': 'test-issuer',
+            'SECRET_KEY': 'test-secret-key'
+        },
+        {
+            'ISSUER': 'https://acme.com/auth',
+            'SECRET_KEY': 'foo',
+        }
+    ],
+
+    'JWT_LEEWAY': 1,
+
+    # NOTE (CCB): This is temporarily set to False until we decide what values are acceptable.
```

Comment:
This comment isn't applicable to this library. The library doesn't care if this option is disabled. It does matter to the IDAs/APIs actually being secured.